cmd/kops/create_cluster.go

```diff
@@ -124,8 +127,17 @@ type CreateClusterOptions struct {
 
 	// ConfigBase is the location where we will store the configuration, it defaults to the state store
 	ConfigBase string
+
+	// DryRun we output data
+	DryRun bool
+
+	// Output type during a DryRun
+	Output string
 }
 
+var targetOptions = sets.NewString(cloudup.TargetCloudformation, cloudup.TargetDirect, cloudup.TargetTerraform)
+var allTargetOptions = strings.Join(targetOptions.List(), ",")
```

Comment:
`targetOptions` is unused?  Can we put direct first, as it is the default?

Reply:
I will remove the set then ...